style.css

```diff
@@ -316,6 +293,181 @@
     border: 1px solid #344563;
   }
 
+  /* Detail Section Starts Here */
+  .no-scroll {
+    overflow: hidden;
+  }
+
+  .blur-modal {
+    filter: blur(3px);
+  }
+
+  .detail-card {
+    display: none;
+  }
+
+  .detail-card--show {
```

Comment:
- [ ] Great job implementing the required changes from the previous reviewer. However, there're few things you still need to work on before your project completely meet the project requirement.
  - Your popup window must contain all the information as we have them on the Figma design. You should be able to style your popup card such that it contains all the information on the card as they're on the figma design. At thi moment, your card does not display all the information. Kindly address that.
  - The layout on your design must match the Figma design.

### Your Design
![your-popup-desktop](https://github.com/sulaimonaa/Portfolio/assets/26597432/526e7432-3549-4ff4-bcb3-41624d33444a)

### Figma Design

![tem-popup-desktop](https://github.com/sulaimonaa/Portfolio/assets/26597432/60d39f0a-619a-4619-b461-9857db3cef29)